src/Composer/ArtifactsPlugin.php

```diff
@@ -42,6 +42,16 @@ public function activate(Composer $composer, IOInterface $io): void
         $this->registerProviders($repository, $composer);
     }
 
+    public function deactivate(Composer $composer, IOInterface $io)
+    {
+        // Nothing to do here
```

Comment:
We should unregister the provider here, shouldn't we?

Reply:
the `deactivate()` method seems to be relevant if one registers a plugin class, so that class is unloaded before a plugin is installed. That's not the case with the `contao/manager-plugin`.